pyvistaqt/plotting.py

```diff
@@ -242,6 +242,7 @@ def __init__(
 
         if off_screen:
             self.ren_win.SetOffScreenRendering(1)
+            self.iren = None
```

Comment:
FYI I think this will help with https://github.com/mne-tools/mne-python/pull/8602, when I was working on that PR I think I almost made a PR to this repo for this change